app/setup_handlers.go

```diff
@@ -30,28 +33,27 @@ func GetDefaultUpgradeHandlerVersion() string {
 }
 
 func SetupHandlers(app *App) {
-	allUpgrades := upgradeTracker{
-		upgrades: []upgradeTrackerItem{
-			// TODO: enable back IBC
-			// these commented lines allow for the IBC modules to be added to the upgrade tracker
-			// https://github.com/zeta-chain/node/issues/2573
-			//{
-			//	index: <CURRENT TIMESTAMP>,
-			//	storeUpgrade: &storetypes.StoreUpgrades{
-			//		Added: []string{
-			//			capabilitytypes.ModuleName,
-			//			ibcexported.ModuleName,
-			//			ibctransfertypes.ModuleName,
-			//		},
-			//	},
-			//},
-			//{
-			//	index: <CURRENT TIMESTAMP>,
-			//	storeUpgrade: &storetypes.StoreUpgrades{
-			//		Added: []string{ibccrosschaintypes.ModuleName},
-			//	},
-			//},
+
+	addErc20ModuleUpgradeTracker := upgradeTrackerItem{
+		index: 1752528615,
+		storeUpgrade: &storetypes.StoreUpgrades{
+			Added: []string{erc20types.ModuleName},
 		},
+	}
+
+	var upgrades []upgradeTrackerItem
+	if app.ChainID() != "" {
+		evmChaindID, err := chains.CosmosToEthChainID(app.ChainID())
+		if err != nil {
+			fmt.Println("Skip here")
```

Comment:
I think we should rather panic, better to have the chain halting than moving into an inconsistent state